src/components/ItaliaTheme/GalleryPreview/GalleryPreview.jsx

```diff
@@ -74,7 +74,8 @@ const GalleryPreview = ({ id, viewIndex, setViewIndex, items }) => {
             closeAriaLabel={intl.formatMessage(messages.close_preview)}
             toggle={closeModal}
           >
-            {items[viewIndex].title}{' '}
+            <h4>{items[viewIndex].title}</h4>
+            <p>{items[viewIndex].description}</p>
```

Comment:
Se la description non c'è va in errore? non servirebbe un controllo per non mostrare comunque il <p>?